### PR TITLE
[react, mui, css] create login and signUp components and style it 

### PR DESCRIPTION
**Pull request by MostafaBalousha123**
Screenshot (27)

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -38,6 +38,8 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
+    <script nomodule src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"></script>
```
Comment (aaamra) [inline client/public/index.html:42]: it's not good to use cdn links for icons and styles with react
you can just install it in your app with `npm i ....`
and Marital UI has a great collection of icons
[MUI Icons](https://mui.com/material-ui/material-icons/)

**Review by aaamra — COMMENTED** [review]
image

i think the `auth` folder should be in the `pages` folder.
not in `components`

**Review by aaamra — COMMENTED** [review]
image

you are using MUI but you did not installed it !?
how it works for you ?

File: client/src/components/auth/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <GoogleLogo width={20} />
+        <p>Sign in with Google</p>
+      </Button>
+
+      <p className="center-pra">
+        Don’t have an account ?
+        <a href="#"> Sign Up!</a>
+      </p>
+    </div>
+    <img src="https://cdn.discordapp.com/attachments/959502807071867000/1029343065720246354/pexels-rodnae-productions-7348675.jpg" alt="test" style={{ margin: '0', padding: '0' }} />
```
Comment (aaamra) [inline client/src/components/auth/Login.tsx:27]: don't use links for your images
you can add it in the `assets` folder

File: client/src/components/auth/SignUp.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        <p>Sign up with Google</p>
+      </Button>
+
+      <p className="center-pra">
+        have an account ?
+        <a href="#"> Sign in!</a>
+      </p>
+    </div>
+
+    <img src="https://cdn.discordapp.com/attachments/959502807071867000/1029343065720246354/pexels-rodnae-productions-7348675.jpg" alt="test" />
```
Comment (aaamra) [inline client/src/components/auth/SignUp.tsx:30]: same issue here
put the image in the `assets` folder

File: client/src/components/auth/auth.css [hunk 4]
```diff
@@ -0,0 +1,82 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap');
```
Comment (aaamra) [inline client/src/components/auth/auth.css:1]: i think if you add this font in a global css file may be better to keep the website with the same font

**Review by Bakeza — DISMISSED** [review]

**Review by shamskhodary — APPROVED** [review]

File: client/package.json [hunk 5]
```diff
@@ -3,16 +3,23 @@
...
+    "@mui/styled-engine-sc": "^5.10.6",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "@types/jest": "^27.5.2",
     "@types/node": "^16.11.64",
     "@types/react": "^18.0.21",
     "@types/react-dom": "^18.0.6",
+    "install": "^0.13.0",
+    "npm": "^8.19.2",
```
Comment (aaamra) [inline client/package.json:18]: why did you installed this ?

File: client/package.json [hunk 6]
```diff
@@ -3,16 +3,23 @@
...
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
+    "styled-components": "^5.3.6",
```
Comment (aaamra) [inline client/package.json:22]: and this

**Review by aaamra — CHANGES_REQUESTED** [review]
add this rule in `.eslintrc.json`
```
"max-len": ["error", { "code": 80 }]
```

File: client/package.json (hunk 5 above)
Comment (MostafaBalousha123) [reply client/package.json:18]: Oh technical error 😢

File: client/package.json (hunk 6 above)
Comment (MostafaBalousha123) [reply client/package.json:22]: this for MUI

**Review by aaamra — APPROVED** [review]

**Review by shamskhodary — APPROVED** [review]

**Review by Bakeza — DISMISSED** [review]

**Review by Bakeza — APPROVED** [review]

**Review by shamskhodary — APPROVED** [review]

**Review by aaamra — APPROVED** [review]

**Pull request merged**